stdlib/format.mli

```diff
@@ -311,6 +311,10 @@ val set_margin : int -> unit
   Nothing happens if [d] is smaller than 2.
   If [d] is too large, the right margin is set to the maximum
   admissible value (which is greater than [10 ^ 9]).
+  If [d] is inferior to the current maximum indentation limit, the
```

Comment:
Not a big deal, but I think that 'less than' would be more widely understood than 'inferior to'.